spring-boot-project/spring-boot-test/src/test/java/org/springframework/boot/test/mock/mockito/SpyBeanWithAopProxyTests.java

```diff
@@ -91,6 +93,8 @@ public Long getDate(boolean arg) {
 			return System.nanoTime();
 		}
 
+		@CacheEvict(value = "test", allEntries = true)
+		public void clearCache() {}
```

Comment:
FTR, if we decide to pursue with this, I am not keen to use the annotation model for this. Rather I'd clean the test `Cache`by an explicit call to its API.